manifests/tuning.pp

```diff
@@ -0,0 +1,20 @@
+# Set ulimits max open files
+#
+define riak::tuning::limits ($user, $type, $item, $value) {
```

Comment:
The parameters should be typed as well, e.g. `(String[1] $user, String[1] $type, String[1]$item, Integer $value)` assuming that these types are correct. If it's possible to have more than one type, we can account for that as well.